delphin/itsdb.py

```diff
@@ -698,7 +698,7 @@ def _split_cols(colstring):
     if not colstring:
         return None
     colstring = colstring.lstrip(':')
-    return list(map(str.strip, colstring.split('@')))
+    return list(map(lambda unistr: unistr.strip(), colstring.split('@')))
```

Comment:
Yes, this is the fix I was thinking of, but a list comprehension might make it a bit clearer:

```python
   return [col.strip() for col in colstring.split('@')]
```